domain/application/modelmigration/import.go

```diff
@@ -46,6 +46,11 @@ type ImportService interface {
 	CreateApplication(context.Context, string, service.AddApplicationParams, ...service.AddUnitParams) error
 }
 
+// Name returns the name of this operation.
+func (i *importOperation) Name() string {
+	return "import applications"
```

Comment:
my only question is all the prefix of import and export to the name string redundant and can be added one level higher?